sdk/java/src/main/java/com/gojek/feast/FeastClient.java

```diff
@@ -52,8 +56,12 @@
    * @return {@link FeastClient}
    */
   public static FeastClient create(String host, int port) {
-    ManagedChannel channel = ManagedChannelBuilder.forAddress(host, port).usePlaintext().build();
-    return new FeastClient(channel, Optional.empty());
+    try {
+      // configure client with no security config.
+      return FeastClient.createSecure(host, port, SecurityConfig.newBuilder().build());
+    } catch (SSLException e) {
+      throw new RuntimeException(e);
```

Comment:
Is there a reason to rethrow without any attached message?

Reply:
Updated `createSecure()` to throw `IllegalArgumentException` when provided with bad certificate.
- Removed redundant try catch in `create()`